File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/TicketDao.java
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api;

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface;
import ru.kami.gis.afisha.schedule.core.domain.TicketEntity;

import java.util.List;

/**
 * @author Daniil.Makarov
 */
public interface TicketDao extends JdbcDaoInterface {
    List<TicketEntity> findAllTicketsByIdEvent(Long eventId);

    int insert(TicketEntity ticketEntity);
}

File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/TicketDao.kt
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface
import ru.kami.gis.afisha.schedule.core.domain.TicketEntity

/**
 * @author Daniil.Makarov
 */
interface TicketDao : JdbcDaoInterface {
    fun findAllTicketsByIdHallAndEvent(hallId: Long, eventId: Long): List<TicketEntity?>
}